Remotion/Web/Development.WebTesting/Accessibility/AccessibilityRequirementID.cs

```diff
@@ -24,78 +24,145 @@ namespace Remotion.Web.Development.WebTesting.Accessibility
   public enum AccessibilityRequirementID
   {
     Unknown,
+
+    // --- Aria Checks ---
     AbstractRole,
     AriaAllowedAttr,
-    AriaHiddenBody,
+    AriaAllowedRole, //New
```

Comment:
Whats the purpose of all the `//New`? Are they just hints for the review? Because we should definitely remove them before merging.